src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -565,12 +565,14 @@ namespace
                     pretty_uint(tile_converged_pixel_count),
                     converged_pixels_string));
             stats.insert("samples/pixel", m_spp);
```

Comment:
A better solution that comes to mind is to insert a string that you generated using `foundation::pretty_scalar()`.

Reply:
Thanks, that worked.